c10/macros/build.bzl

```diff
@@ -29,3 +29,12 @@ def define_targets(rules):
             "//conditions:default": [],
         }),
     )
+    rules.filegroup(
+        name = "headers",
+        srcs = rules.glob(
+            ["*.h"],
+            exclude = [
```

Comment:
I think the empty `exclude` [can be omitted.](https://bazel.build/reference/be/functions#glob_example)

Reply:
Perhaps, but this is a copy-n-paste from https://github.com/pytorch/pytorch/blob/177621a0b28b931d9be6976c2c38cb57af7949d9/c10/util/build.bzl#L66-L70